.circleci/config.yml

```diff
@@ -83,19 +94,23 @@ jobs:
       - run:
           name: Setup WordPress testing framework
           command: |
-            /bin/bash tests/install-wp-tests.sh wpe_headless_tests wpe_headless_user wpe_headless_pass 127.0.0.1 latest true
+            /bin/bash tests/install-wp-tests.sh wp_database wp_user wp_pass 127.0.0.1 latest true
           working_directory: .
       - run:
           name: Run testing suite
           command: |
             composer test
+            rm composer-setup.php
```

Comment:
@mindctrl removing the composer-setup.php, which was installed during 'php/install-composer'. Doing after tests run so it's not left on the file system and added to the zip.  Good catch.